product_docs/docs/tde/15/secure_key/key_store.mdx

```diff
@@ -92,4 +93,8 @@ See the [Using](/partner_docs/ThalesCipherTrustManager/05-UsingThalesCipherTrust
 
 ## Fortanix Data Security Manager example 
 
-See [Using Fortanix Data Security Manager with EDB Postgres for TDE](https://support.fortanix.com/docs/using-fortanix-data-security-manager-with-edb-postgres-for-tde) for a step-by-step configuration tutorial.
\ No newline at end of file
+See [Using Fortanix Data Security Manager with EDB Postgres for TDE](https://support.fortanix.com/docs/using-fortanix-data-security-manager-with-edb-postgres-for-tde) for a step-by-step configuration tutorial.
+
+## Entrust KeyControl 
```

Comment:
Should this be example to match the previous headings (or are the previous headings incorrectly labelled example?)